docs/topics/whatsnew15.md

```diff
@@ -0,0 +1,740 @@
+They map the original collection with the custom selector function and return the first non-null value. If there is no such value,
+`firstNotNullOf()` throws an exception, and `firstNotNullOfOrNull()` returns null.
+
+```kotlin
+fun main() {
+//sampleStart
+    val data = listOf("Kotlin", "1.5")
+    println(data.firstNotNullOf(String::toDoubleOrNull))
+    println(data.firstNotNullOfOrNull(String::toIntOrNull))
+//sampleEnd
+}
+```
+
+### Strict version of String?.toBoolean()
+
+Two new functions introduce case-sensitive strict versions of the existing [String?.toBoolean()](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/to-boolean.html):
+* `String.toBooleanStrict()`<!--(TODO link)-->  throws an exception for all inputs except the literals `true` and `false`.
+* `String.toBooleanStrictOrNull()`<!--(TODO link)-->  returns null for all inputs except the literals `true` and `false`.
+
+```kotlin
+fun main() {
+//sampleStart
+    println("true".toBooleanStrict())
+    println("1".toBooleanStrictOrNull())
+    println("1".toBooleanStrict())
+//sampleEnd    
+}
+```
+
+## kotlin-test library
+The [kotlin-test](https://kotlinlang.org/api/latest/kotlin.test/) library introduces some new features:
+* [Simplified test dependencies usage in multiplatform projects](#simplified-test-dependencies-usage-in-multiplatform-projects)
+* [Automatic selection of a testing framework for Kotlin/JVM source sets](#automatic-selection-of-a-testing-framework-for-kotlin-jvm-source-sets)
+* [Assertion function updates](#assertion-function-updates)
+
+### Simplified test dependencies usage in multiplatform projects
+
+Now you can use the `kotlin-test` dependency to add dependencies for testing in the `commonTest` source set, and the Gradle plugin
+will infer the corresponding platform dependencies for each source set:
+* `kotlin-test-junit`/`kotlin-test-junit-5`/`kotlin-test-testng` for JVM source sets, see [automatic choice of a testing framework for Kotlin/JVM source sets](#automatic-selection-of-a-testing-framework-for-kotlin-jvm-source-sets)
+* `kotlin-test-js` for Kotlin/JS source sets
+* `kotlin-test-common` and `kotlin-test-annotations-common` for common source sets
+* No extra artifact for Kotlin/Native source sets
+
+Additionally, you can use the `kotlin-test` dependency in any shared or platform-specific source set.
+
+An existing kotlin-test setup with explicit dependencies will continue to work both in Gradle and in Maven.
+
+Learn more about [setting dependencies on test libraries](gradle.md#set-dependencies-on-test-libraries).
+
+### Automatic selection of a testing framework for Kotlin/JVM source sets
+
+The Gradle plugin now chooses and adds a dependency on a testing framework automatically. All you need to do is add
+the dependency `kotlin-test` in the common source set.
+
+Gradle uses JUnit 4 by default; therefore, the `kotlin('test')` dependency resolves to the variant for JUnit 4, namely `kotlin-test-junit`:
```

Comment:
Should ```variant``` be removed here?

Reply:
Yes, seems that it doesn't add value here.